docker/docker-compose.yml

```diff
@@ -80,7 +80,8 @@ services:
     environment:
       - MYSQL_ROOT_PASSWORD=root
       - MYSQL_DATABASE=tests
-      - MYSQL_USER=prisma
+      - MYSQL_USER=prisma # not used
```

Comment:
Should we keep this if we always use the root user?

Reply:
I don't know, honestly. 
This not being used might actually be a smell we want to fix (in the future)?

Reply:
It's fine to not do anything about this